src/survey-config/dto/response-survey-config.dto.ts

```diff
@@ -1,13 +1,26 @@
 import { TimeUnitsEnum } from "@prisma/client";
-import { ResponseSurveyCycleParameterDto } from "src/survey-cycle-parameter/dto/response-survey-cycle-parameter.dto";
 
 export class ResponseSurveyConfigDto {
   readonly id: number;
-  readonly departmentId: number;
+  readonly surveyName: string;
   readonly onboardingTime: number;
   readonly onboardingTimeUnit: TimeUnitsEnum;
   readonly startTime: Date;
   readonly endTime: Date;
   readonly isActive: boolean;
-  readonly SurveyCycleParameters?: ResponseSurveyCycleParameterDto[];
+}
+
+export class ConfigResponseDTO {
+  readonly data: ResponseSurveyConfigDto;
+  readonly message: string;
+}
+
+export class UserMappingDTO {
+  assesseeId: string;
+  assessorIds: string;
```

Comment:
array of string ID for assessorIds

Reply:
This DTO is used for parsing the CSV data where it won't be an array of strings.